mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -24,7 +24,7 @@
 from mlos_bench.services.base_service import Service
 from mlos_bench.services.types.config_loader_type import SupportsConfigLoading
 from mlos_bench.tunables.tunable_groups import TunableGroups
-from mlos_bench.util import instantiate_from_config, path_join, BaseTypeVar
+from mlos_bench.util import merge_parameters, instantiate_from_config, path_join, BaseTypeVar
```

Comment:
```suggestion
from mlos_bench.util import instantiate_from_config, merge_parameters, path_join, BaseTypeVar
```

nit: keeps it more sorted and matching the style of the one above as well